deconz/data/run.sh

```diff
@@ -24,6 +24,7 @@ VNC_PASSWORD=$(bashio::config 'vnc_password')
 WEBSOCKET_PORT=$(bashio::addon.port 8080)
 
 # Lookup udev link
+sleep 3
```

Comment:
Isn't it better to add a loop here with a timeout? To check if it is available, and continue immediately is so?

Reply:
Or do a while link does not exists counter++, sleep 1; if max counter -> bashio::exit.nok "timeout reached"

Reply:
If the device does not exist on that point, he will never exist. Because it's docker, the device needs plugin before the add-on start. The sleep is not really needed, only a small helper for slower device.

Reply:
Sure! totally understand, my suggestion was to make the sleep conditional using a loop, so faster devices will stall for 3 seconds.